src/ArrowEffects.cpp

```diff
@@ -736,20 +741,10 @@ float ArrowEffects::GetXPos( const PlayerState* pPlayerState, int iColNum, float
 			fEffects[PlayerOptions::EFFECT_TAN_BUMPY_X_PERIOD]), curr_options->m_bCosecant );
 
 	if( fEffects[PlayerOptions::EFFECT_DRUNK] != 0 )
-		fPixelOffsetFromCenter += fEffects[PlayerOptions::EFFECT_DRUNK] *
-			( RageFastCos( CalculateDrunkAngle(fEffects[PlayerOptions::EFFECT_DRUNK_SPEED], iColNum,
-					fEffects[PlayerOptions::EFFECT_DRUNK_OFFSET], DRUNK_COLUMN_FREQUENCY,
-					fYOffset, fEffects[PlayerOptions::EFFECT_DRUNK_PERIOD],
-					DRUNK_OFFSET_FREQUENCY) ) * ARROW_SIZE*DRUNK_ARROW_MAGNITUDE );
-
-	if( fEffects[PlayerOptions::EFFECT_TAN_DRUNK] != 0 )
-		fPixelOffsetFromCenter += fEffects[PlayerOptions::EFFECT_TAN_DRUNK] *
-			( SelectTanType( CalculateDrunkAngle(fEffects[PlayerOptions::EFFECT_TAN_DRUNK_SPEED],
-					iColNum, fEffects[PlayerOptions::EFFECT_TAN_DRUNK_OFFSET],
-					DRUNK_COLUMN_FREQUENCY, fYOffset,
-					fEffects[PlayerOptions::EFFECT_TAN_DRUNK_PERIOD], DRUNK_OFFSET_FREQUENCY)
-					, curr_options->m_bCosecant) * ARROW_SIZE*DRUNK_ARROW_MAGNITUDE );
-
+		fPixelOffsetFromCenter += fEffects[PlayerOptions::EFFECT_DRUNK] * 
+			( std::cos( RageTimer::GetTimeSinceStartFast() + iColNum*DRUNK_COLUMN_FREQUENCY
```

Comment:
Why is this so different? Keep stuff the same and only do what the PR says it's doing.